### DISCUSSION TITLE
version 1.0

### DISCUSSION THREAD

**Discussion by Ihsanullah-Yasar** [skillsupp/Teampartifilio#46] [Announcements]
### version 1.0
@skillsupp/frontend 
@skillsupp/backend

## What's Changed
* hi work team by @SanaullahAnwari in https://github.com/skillsupp/Teampartifilio/pull/27

## New Contributors
* @SanaullahAnwari made their first contribution in https://github.com/skillsupp/Teampartifilio/pull/27

**Full Changelog**: https://github.com/skillsupp/Teampartifilio/commits/v1

<hr /><em>This discussion was created from the release <a href='https://github.com/skillsupp/Teampartifilio/releases/tag/v1'>version 1.0</a>.</em>

**Comment by SharifullahCS** [comment]
What are u done in this release? I don't know what is it?
